mobile-widgets/qml/DiveSummary.qml

```diff
@@ -0,0 +1,210 @@
+// SPDX-License-Identifier: GPL-2.0
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Controls 1.4
+import QtQuick.Layouts 1.12
+import QtQuick.Dialogs 1.3
+import org.subsurfacedivelog.mobile 1.0
+import org.kde.kirigami 2.4 as Kirigami
+
+Kirigami.ScrollablePage {
+	background: Rectangle { color: subsurfaceTheme.backgroundColor }
+	title: qsTr("Dive summary")
+
+	ListModel {
+		id: monthModel
+		ListElement {text: qsTr("Total")}
+		ListElement {text: qsTr(" 1 month")}
+		ListElement {text: qsTr(" 2 month")}
+		ListElement {text: qsTr(" 3 month")}
+		ListElement {text: qsTr(" 4 month")}
+		ListElement {text: qsTr(" 5 month")}
+		ListElement {text: qsTr(" 6 month")}
+		ListElement {text: qsTr(" 7 month")}
+		ListElement {text: qsTr(" 8 month")}
+		ListElement {text: qsTr(" 9 month")}
+		ListElement {text: qsTr("10 month")}
+		ListElement {text: qsTr("11 month")}
+		ListElement {text: qsTr("12 month")}
+	}
+
+	onVisibleChanged: {
+		if (visible)
+			Backend.summaryCalculation(selectionPrimary.currentIndex, selectionSecondary.currentIndex)
+	}
+
+	GridLayout {
+		columns: 3
+		width: parent.width
+		columnSpacing: Kirigami.Units.smallSpacing
+		rowSpacing: Kirigami.Units.smallSpacing
+
+		TemplateLabel {
+			text: qsTr("oldest/newest dive")
+			font.bold: true
+		}
+		TemplateLabel {
+			text: Backend.resultCalculation[0]
```

Comment:
it is not elegant for sure, but I wanted to avoid 20+ names. I have been thinking of using a named list (think json), but decided against it due to the added complexity.

Reply:
yeah, as I said, I think this is the easiest solution here.
I briefly thought "well, you could create named properties in QML and assign the values to them and then use those properties"... that would make it a tiny bit more readable, but I'm not sure it's worth the effort.
I have a few minutes right now, let me look at this locally.